exercises/printUserInfo.js

```diff
@@ -26,17 +30,10 @@ console.log();
 
 console.log(`How's the weather in ${user['currentLocation']}?`);
 console.log();
+console.log(user['favoriteMovies']);
+console.log(printFavorites(user['favoriteMovies']));
```

Comment:
You're calling `printFavorites` here with a single argument, but you defined it to accept no arguments.